pkg/gce-pd-csi-driver/gce-pd-driver_test.go

```diff
@@ -54,14 +54,28 @@ func controllerServerForTest(cloudProvider gce.GCECompute) *GCEControllerServer
 		SupportsIopsChange:       []string{"hyperdisk-balanced", "hyperdisk-extreme"},
 		SupportsThroughputChange: []string{"hyperdisk-balanced", "hyperdisk-throughput", "hyperdisk-ml"},
 	}
-
-	return NewControllerServer(gceDriver, cloudProvider, errorBackoffInitialDuration, errorBackoffMaxDuration, fallbackRequisiteZones, enableStoragePools, enableDataCache, multiZoneVolumeHandleConfig, listVolumesConfig, provisionableDisksConfig, true /* enableHdHA */)
+	return NewControllerServer(gceDriver, cloudProvider, errorBackoffInitialDuration, errorBackoffMaxDuration, fallbackRequisiteZones, enableStoragePools, enableDataCache, multiZoneVolumeHandleConfig, listVolumesConfig, provisionableDisksConfig, true /* enableHdHA */, args)
 }
 
-func initGCEDriverWithCloudProvider(t *testing.T, cloudProvider gce.GCECompute) *GCEDriver {
+func initGCEDriverWithCloudProvider(t *testing.T, cloudProvider gce.GCECompute, args ...*GCEControllerServerArgs) *GCEDriver {
 	vendorVersion := "test-vendor"
 	gceDriver := GetGCEDriver()
-	controllerServer := controllerServerForTest(cloudProvider)
+
+	// Passing args as a variadic argument prevents updating many function
```

Comment:
This comment won't make any sense once this PR is updated... there's only a dozen call sites, let's just update them all now  as initGCEDriverWithCloudProvider(t, provider, GCEControllerServerArgs{}).

It will simplify the logic below quite a bit which is a win for future generations.